### ISSUE TITLE
Include sub-directory when packaging the release ZIP

### ISSUE THREAD

**Issue by AlexanderWert** [grafana/influxdb-flux-datasource#30]
When using Grafana's GF_INSTALL_PLUGINS property for installing this datasource when using the grafana docker image ([http://docs.grafana.org/installation/docker/](http://docs.grafana.org/installation/docker/)), the installation fails.

**Example**:
GF_INSTALL_PLUGINS=https://github.com/grafana/influxdb-flux-datasource/releases/download/5.2.11/influxdb-flux-datasource-5.2.11.zip;Flux

**Reason that it fails:**
The release ZIP-file does not include a sub-directory. So Grafana extracts all the files directly into the "/var/libs/grafana/plugin" directory and, thus, cannot recognize the datasource correctly.

It would be great if all the content within the release ZIP files could be moved to a sub-directory of the zip-file, so this great Grafana installation feature can be used!

Thanks!

**Comment by briangann** [comment]
that was a change to the new toolkit package, and will be updated to include the path in a future release.

**Comment by amotl** [comment]
Hi there,

we had the very same issue which has been resolved through https://github.com/panodata/grafana-map-panel/issues/15 and https://github.com/grafana/grafana/issues/20071 ff., thanks to @dprokop and @ryantxu.

We just checked with the current master and can confirm the packaging now includes the subdirectory as intended.

```
npx grafana-toolkit plugin:ci-build
npx grafana-toolkit plugin:ci-package
```

So, closing this issue might be appropriate.

With kind regards,
Andreas.
